README.md

```diff
@@ -35,7 +35,13 @@ Installation is straightforward.  Pwndbg is best supported on Ubuntu 18.04 with
 ```shell
 git clone https://github.com/pwndbg/pwndbg
 cd pwndbg
-./setup.sh
+./setup.sh virtualenv
```

Comment:
Maybe:
```
# if you want to use virtualenv pass in the 'virtualenv' first argument:
./setup.sh virtualenv
# otherwise, just call
./setup.sh
```

Reply:
Or even adding "(requires virtualenv to be installed)"

Reply:
Yes but indeed it will make the install instructions hard to read. I do not really like that. I really enjoyed the simplicity to install pwndbg when I installed it for the first time.

Reply:
https://github.com/pwndbg/pwndbg/pull/941#discussion_r686627137 => virtualenv is now installed from repo on any OS.

Reply:
The default should NOT be to use a virtualenv.   This should be a supplementary option, perhaps in a subsection.